skycultures/indian/index.json

```diff
@@ -737,49 +737,49 @@
 	      "link": { "star": 65474, "offset": 180},
 	      "comment": "Please replace by Indian names, and remove symbols if not used traditionally!",
 	      "names": [ { "symbol": "\u2648", "native": "Aries", "english": "Ram"},
-			 { "symbol": "\u2649", "native": "Taurus", "english": "Bull"},
-			 { "symbol": "\u264A", "native": "Gemini", "english": "Twins"},
-			 { "symbol": "\u264B", "native": "Cancer", "english": "Crab"},
-			 { "symbol": "\u264C", "native": "Leo", "english": "Lion"},
-			 { "symbol": "\u264D", "native": "Virgo", "english": "Virgin"},
-			 { "symbol": "\u264E", "native": "Libra", "english": "Scales"},
-			 { "symbol": "\u264F", "native": "Scorpius", "english": "Scorpion"},
-			 { "symbol": "\u2650", "native": "Sagittarius", "english": "Archer"},
-			 { "symbol": "\u2651", "native": "Capricornus", "english": "Capricorn"},
-			 { "symbol": "\u2652", "native": "Aquarius", "english": "Water bearer"},
-			 { "symbol": "\u2653", "native": "Pisces", "english": "Fishes"}]},
+			 { "symbol": "\u2649", "native": "Vr̥ṣabha", "english": "Bull"},
```

Comment:
Ah, maybe you did not read the latest developments since 25.1. We encourage culture-native terms now in the "native" tags, and latin-letter-with-diacritics transliteration in the "pronounce" tag. So, what you now added under "native" should be "pronounce", and Sanskrit words should be in "native". If the actual translation of the figures differ (I have read somewhere, twins may be ashvini/horsemen?, maybe others), please modify the English word to provide the translation of the actual Indian meaning.

Reply:
Done. 
Twins is ok.